src/lang/en.json

```diff
@@ -863,7 +863,7 @@
     "Remote Browser": "Remote Browser",
     "Add a Remote Browser": "Add a Remote Browser",
     "Remote Browser not found!": "Remote Browser not found!",
-    "remoteBrowsersDescription": "Remote Browsers are an alternative to running Chromium locally. Setup with a service like browserless.io or connect to your self-hosted container: https://hub.docker.com/r/browserless/chrome",
```

Comment:
I think my last comment may be not clear, the url is good to have, but it should be in <a></a> and no need to be translatable.

Reply:
Here are the docs for component interpolation (which is how you can integrate links into translation without resoring to `v-html`): https://vue-i18n.intlify.dev/guide/advanced/component

Reply:
@CommanderStorm Actually we don't need this, because translating a url is meaningless. What I meant is that we just need to put the raw url in the vue component, no need to be in i18n.

(Off topic, I just sent a email to you hope you can receive it)

Reply:
@louislam We are talking past each other, component translation or utilising `v-html` (not recommended, this would allow our translators to XSS us) is necessary because of vue's html-escaping.

I did not suggest translating the url ^^

example (not using the slot syntax, unshure which one is better):

- vue:
  ```
  <i18n-t keypath="remoteBrowsersDescription">
    <a href="https://browserless.io" target="_blank">browserless.io</a>
    <a href="https://hub.docker.com/r/browserless/chrome" target="_blank">{{ $t('self-hosted container') }}</a>
  </i18n-t>
  ```
- `en.json`:
  ```diff
  - "remoteBrowsersDescription": "Remote Browsers are an alternative to running Chromium locally. Setup with a service like browserless.io or connect to your self-hosted container: https://hub.docker.com/r/browserless/chrome",
  + "remoteBrowsersDescription": "Remote Browsers are an alternative to running Chromium locally. Setup with a service like {0} or connect to your {0}",
  + "self-hosted container": "self-hosted container",
  ```

Reply:
> utilising v-html (not recommended, this would allow our translators to XSS us

Wait is this actually possible? There are currently a few places where we do use `v-html` for translation 🤔

Reply:
> Wait is this actually possible? 

Yes, it should accept `<script></script>`. Bad translators should be able to add it via Weblate, but usually I will have a quick look before merge those translation PRs (#3891). So it won't be a big issue I think.

Reply:
I've updated the translation to the use the same as @CommanderStorm's example in en.json